api/pom.xml

```diff
@@ -271,22 +218,22 @@
                 </configuration>
                 <executions>
                     <execution>
-                       <id>attach-sources</id>
-                       <goals>
-                           <goal>jar-no-fork</goal>
-                       </goals>
+                        <id>attach-sources</id>
+                        <goals>
+                            <goal>jar-no-fork</goal>
+                        </goals>
                     </execution>
                 </executions>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.8</version>
                 <configuration>
-                    <additionalparam>-Xdoclint:none</additionalparam>
+                    <source>1.8</source>
+                    <doclint>none</doclint>
                     <bottom>
-                    <![CDATA[
-                    <p align="left">Copyright &#169; 2019 Eclipse Foundation.<br>Use is subject to
+                        <![CDATA[
+                    <p align="left">Copyright &#169; 2019-${current.year} Eclipse Foundation.<br>Use is subject to
```

Comment:
Does this ${current.year} actually work?  When I experimented with this automatic update, it didn't work for me.  Please verify.  Thanks.

Reply:
Ahh, yes... I see you are using the build-helper-maven-plugin which creates this variable for you.  My mistake.